### ISSUE TITLE
createdAt and updatedAt incremented if migrate is set to alter or drop

### ISSUE THREAD

**Issue by andriussev** [balderdashy/sails#2924]
**Conditions:**
I am using the mysql adapter for my database.
I select a non-default timezone such as 'utc' in the adapter attributes. 
Use 'alter' as my migration attribute in models.

**What happens:**
All of the createdAt and updatedAt columns get incremented every time I restart the application.

**What's expected:**
The records are reentered as they were without redoing the timezone stuff.

EDIT: This probably happens to all datetime fields but I did not check that.

**Comment by sailsbot** [comment]
Thanks for posting, @andriussev.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
